web/src/main/webResources/WEB-INF/config-spring-geonetwork.xml

```diff
@@ -185,4 +185,33 @@
     <constructor-arg name="langCode" value="eng"/>
   </bean>
   -->
+
+  <bean id="gnBackgroundJobScheduler" class="org.springframework.scheduling.quartz.SchedulerFactoryBean">
```

Comment:
We already have one scheduler here https://github.com/geonetwork/core-geonetwork/blob/develop/core/src/main/resources/config-spring-geonetwork.xml#L118-L127 . Can we add that trigger to the list ?

Reply:
Don't know as it's defined in core and i am in services ..

Reply:
services depends on core so it should be fine no ?